src/shared/components/workbasket-filters/workbasket-filters.component.ts

```diff
@@ -320,18 +320,16 @@ export class WorkbasketFiltersComponent implements OnInit {
 
     // Form group local storage is available and contains regionList property
     if (isDefined(formGroupLS) && formGroupLS.hasOwnProperty(REGION_LIST_AND_FRC_FILTER)) {
-      if (this.formGroup.get(REGION_LIST_AND_FRC_FILTER)) {
-        // If regionList value does not match between local storage and form group
-        // then the filter value has been changed and we need to clear the old filter values
-        if (formGroupLS[REGION_LIST_AND_FRC_FILTER] !== this.formGroup.get(REGION_LIST_AND_FRC_FILTER).value) {
-          for (const key in formGroupLS) {
-            if (formGroupLS.hasOwnProperty(key)) {
-              const value = formGroupLS[key];
-              // Clear the filter form group control values if it has a value in local storage
-              // The regionList form group control value should be ignored as it always contain the latest value
-              if (key !== REGION_LIST_AND_FRC_FILTER && value != null) {
-                this.formGroup.get(key).setValue(null);
-              }
+      // If regionList value does not match between local storage and form group
```

Comment:
Do we need to revert this? I remember Lee did this fix when working on a bug.

Reply:
@johnbenjamin-hmcts I have reverted back to the version which is as per revert-to-4.8.3-no-refunds-rc.1